src/main/java/secondstep/WinnerLotto.java

```diff
@@ -0,0 +1,78 @@
+package secondstep;
+
+import java.util.List;
+
+public class WinnerLotto {
+    public static final int LOTTO_NUMBER_THREE_MATCH = 3;
+    public static final int LOTTO_NUMBER_FOUR_MATCH = 4;
+    public static final int LOTTO_NUMBER_FIVE_MATCH = 5;
+    public static final int LOTTO_NUMBER_SIX_MATCH = 6;
+    private static final int FIRST_REWARD = 2000000000;
+    private static final int SECOND_REWARD = 1500000;
```

Comment:
천단위에 언더바를 추가하면 숫자에 대한 가독성도 높일 수 있습니다 :)
```suggestion
    private static final int SECOND_REWARD = 1_500_000;
```